.github/workflows/static.yml

```diff
@@ -0,0 +1,13 @@
+#on: [push, pull_request]
+#name: Static analysis
+
+#jobs:
+#  php-cs-fixer:
+#    name: PHP-CS-Fixer
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
```

Comment:
I recommend you do uncomment these and run PHP-cs-fixer.

I recommend you to do that in another PR.

Reply:
I did it and got some errors - https://gist.github.com/nebkam/adcfdadbcd03e3b90858c6f98e6892c4
Got any hints? I've used PHPStan for static analysis before, but a dockerized PHP CS Fixer is a first for me

Reply:
See #8